tests/e2e/csi_snapshot_basic.go

```diff
@@ -3152,11 +3200,18 @@ var _ = ginkgo.Describe("Volume Snapshot Basic Test", func() {
 
 		labelsMap := make(map[string]string)
 		labelsMap["app"] = "test"
-
 		ginkgo.By("Creating a Deployment using pvc1")
-		dep, err := createDeployment(ctx, client, 1, labelsMap, nil, namespace,
-			[]*v1.PersistentVolumeClaim{pvclaim}, execRWXCommandPod1, false, busyBoxImageOnGcr)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		var dep *appsv1.Deployment
```

Comment:
Please add valid variable name. var deb is not a good name. Also, Declaration should happen before